source/common/common/utility.h

```diff
@@ -561,6 +561,11 @@ template <typename Value> class IntervalSetImpl : public IntervalSet<Value> {
     intervals_.insert(Interval(left, right));
   }
 
+  bool test(Value value) const override {
+    const auto left_pos = intervals_.lower_bound(Interval(value, value + 1));
```

Comment:
Is this assuming an integer representation of intervals? Just trying to figure out why the `+1` works.